workflow/rules/mapping.smk

```diff
@@ -27,22 +27,9 @@ rule bwa_mem:
     wrapper:
         "0.52.0/bio/bwa/mem"
 
-rule mark_duplicates:
-    input:
-        "results/mapped/{sample}-{unit}.bam"
-    output:
-        bam=temp("results/mapped/dedup/{sample}-{unit}.bam"),
-        metrics="results/mapped/dedup/{sample}-{unit}.metrics.txt"
-    log:
-        "logs/picard/dedup/{sample}-{unit}.log"
-    params:
-        "REMOVE_DUPLICATES=true"
-    wrapper:
-        "0.55.1/bio/picard/markduplicates"
-
 rule merge_bams:
     input:
-        lambda w: expand("results/mapped/dedup/{sample}-{unit}.bam",
+        lambda w: expand("results/merged/{sample}-{unit}.bam",
```

Comment:
Was this change intended? Shouldn't this simply be reduced to  `results/mapped/{sample}-{unit}.bam`?